### ISSUE TITLE
Cannot run evaluation using tf object_detection API

### ISSUE THREAD

**Issue by CCThompson82** [CCThompson82/nuclei#39]
evaluation requires correct mounting of cocotools

**Comment by CCThompson82** [comment]
[helpful tutorial](https://towardsdatascience.com/how-to-train-your-own-object-detector-with-tensorflows-object-detector-api-bec72ecfe1d9)

**Comment by CCThompson82** [comment]
After resolving #36, the only changes required updating the model.config file.  The file from trusting_jennings/exp3 is functional and allows evaluation of the validation set.  Attached are sample prediction images after 4439 global steps, acheiving a mAP of 0.4001.

![image](https://user-images.githubusercontent.com/17596748/38144464-b3acfc0c-343c-11e8-91ea-aff339ecc918.png)

![image](https://user-images.githubusercontent.com/17596748/38144527-fab8ffce-343c-11e8-8641-e0acbebdba27.png)

**Comment by CCThompson82** [comment]
Resolved by untracked changes in the configuration file after #36.
